packages/server/database/migrations/20210527170505-addAdditionalRetroTemplates.ts

```diff
@@ -0,0 +1,199 @@
+    groupColor: '#7373E5',
+    templateId: 'dropAddKeepImproveDAKITemplate',
+    sortOrder: 3
+  },
+  {
+    question: 'To Discuss',
+    description: `What should we talk about? We'll group related items into topics and vote on them to decide what to chat about`,
+    groupColor: '#45E5E5',
+    templateId: 'leanCoffeeTemplate',
+    sortOrder: 0
+  },
+  {
+    question: 'Keep Doing 🌀',
+    description: `What behaviors are working and adding value?`,
+    groupColor: '#7373E5',
+    templateId: 'starfishTemplate',
+    sortOrder: 0
+  },
+  {
+    question: 'Less of ➖',
+    description: `What are we overdoing or spending too much time on?`,
+    groupColor: '#D9D916',
+    templateId: 'starfishTemplate',
+    sortOrder: 1
+  },
+  {
+    question: 'More of ➕',
+    description: `What should we take advantage of or spend more time on?`,
+    groupColor: '#45E5E5',
+    templateId: 'starfishTemplate',
+    sortOrder: 2
+  },
+  {
+    question: 'Stop ⏹️',
+    description: `What isn't adding value or helping the team?`,
+    groupColor: '#E55C5C',
+    templateId: 'starfishTemplate',
+    sortOrder: 3
+  },
+  {
+    question: 'Start ⏯️',
+    description: `What new things should we explore or start doing?`,
+    groupColor: '#52CC52',
+    templateId: 'starfishTemplate',
+    sortOrder: 4
+  },
+  {
+    question: 'What went well 😄',
+    description: `What good things happened this sprint?`,
+    groupColor: '#7373E5',
+    templateId: 'whatWentWellTemplate',
+    sortOrder: 0
+  },
+  {
+    question: `What didn't go well 😞`,
+    description: `What didn't go as planned or desired this sprint?`,
+    groupColor: '#E55C5C',
+    templateId: 'whatWentWellTemplate',
+    sortOrder: 1
+  }
+] as const
+
+const templates = templateNames.map((templateName) => makeTemplate(templateName))
+const reflectPrompts = promptsInfo.map((promptInfo: PromptInfo) => makePrompt(promptInfo))
+
+export const up = async function(r: R) {
+  try {
+    await Promise.all([
+      r
+        .table('MeetingTemplate')
+        .insert(templates)
+        .run(),
+      r
+        .table('ReflectPrompt')
+        .insert(reflectPrompts)
+        .run(),
+      // we cache the templates but get the prompts from the db
+      // clear the cache to prevent empty templates
+      redis.flushall()
+    ])
+    redis.disconnect()
+  } catch (e) {
+    console.log(e)
+  }
+}
+
+export const down = async function(r: R) {
+  const templateIds = templates.map(({id}) => id)
+  const promptIds = reflectPrompts.map(({id}) => id)
+  try {
+    await Promise.all([
+      r
+        .table('MeetingTemplate')
+        .getAll(r.args(templateIds))
+        .delete()
+        .run(),
+      r
+        .table('ReflectPrompt')
+        .getAll(r.args(promptIds))
+        .delete()
+        .run(),
+      // we cache the templates but get the prompts from the db
+      // clear the cache to prevent empty templates
+      redis.flushall()
```

Comment:
Nice catch! If we ever did do a down migration, i don't think we'd want to flush the *entire* cache, since we store other tables (e.g. User table) in there.

Can you make it so that we're only deleting the redis keys corresponding to the templates created here? An example is in [`endSprintPoker`](https://github.com/ParabolInc/parabol/blob/master/packages/server/graphql/mutations/endSprintPoker.ts#L58-L61). It looks like we just [prefix keys with table name](https://github.com/ParabolInc/parabol/blob/master/packages/server/dataloader/LocalCache.ts#L126).

Reply:
Good point! I've pushed an update removing the public templates key